### PR TITLE
Page Speed

### PR DESCRIPTION
**Pull request by danielcj2**
Will get the fields hooked up to local state in my next PR. 

Issues: 
 - proptype error. Need to refactor `StatusLabel` component to allow for different status texts

### CODE REVIEW

File: Client/src/Utils/timeUtils.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+export const formatDuration = (ms) => {
```
Comment (ajhollid) [inline Client/src/Utils/timeUtils.js:1]: Nice, thanks for refactoring this out

File: Client/src/Pages/PageSpeed/index.jsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  return (
+    <Grid item lg={6} flexGrow={1}>
+      <Stack direction="row" gap={theme.gap.medium} p={theme.gap.ml}>
+        <PageSpeedIcon style={{ width: theme.gap.ml, height: theme.gap.ml }} />
+        <Box flex={1}>
+          <Stack direction="row" justifyContent="space-between">
+            <Typography component="h2" mb={theme.gap.xs}>
+              {data.name}
+            </Typography>
+            {/* TODO - fix prop-type error */}
```
Comment (ajhollid) [inline Client/src/Pages/PageSpeed/index.jsx:40]: We could just remove the acceptable `strings` from the label component, I don't think there's any real good reason to limit what strings we can use there :thinking:

**Review by ajhollid — APPROVED** [review]
Nice work, appreciate `timeUtils` refactor :100:

File: Client/src/Pages/PageSpeed/index.jsx (hunk 2 above)
Comment (danielcj2) [reply Client/src/Pages/PageSpeed/index.jsx:40]: Yea, that's probably the move. I'm thinking we should define some styles types in the component itself as well, that way there's no need to pass a color variable for background/dot at all.

**Review by Skorpios604 — APPROVED** [review]
A masterpiece

**Pull request merged**